### ISSUE TITLE
Cannot resolve to SCSS partial files

### ISSUE THREAD

**Issue by hinok** [mrmlnc/vscode-scss#39]
Hey,
I've found that when trying to import [scss partial file](http://sass-lang.com/guide#topic-4), plugin doesn't work correctly because it doesn't resolve to scss partial file with leading underscore e.g. `_partial.scss`.

In dev tools
```javascript
{
  "errno":-2,
  "code":"ENOENT",
  "syscall":"stat",
  "path":"/Users/Batman/Desktop/Wayne/src/index.scss" 
  // index.scss doesn't exist but this exists _index.scss
}
```

When trying to resolve path for `@import '...'`, I think that plugin should also check if partial file exists, like

```
/Users/Batman/Desktop/Wayne/src/_index.scss
```

**Comment by Arikael** [comment]
I just wanted to add that if you don't import your partial files including underscore and extension, the whole plugins crashes and stops working even in unrelated files.

so if you, like me always import partials without extension or underscore and the extension is not working this may be the cause.

Also: I had to include // <reference path="./[myfile]" />

**Comment by Christo44** [comment]
Ah this might be what I'm experiencing too.

@Arikael can you give me an example of how you structure your paths ?

**Comment by mrmlnc** [comment]
Will be cool, if you give me more examples of your problem. Now my [fast & lightweight parser]() has no support for `@include font-face("Akkurat-Bold", "../fonts/akkurat/lineto-akkurat-bold");`. I try to find time for this issue and #42.

**Comment by Arikael** [comment]
@Christo44 
Sorry, we recently switched from VS Code to WebStorm and I haven't used VS Code since then.
If I remember correctly I had to explicitly import any partials including underscore and extensions

  `@import '_bourbon.scss;`

instead of 
`@import 'bourbon'`

**Comment by Christo44** [comment]
@Arikael 
using `@import 'bourbon'` gives no errors in vscode, but using `@import '_bourbon.scss'` does.

```
[Error - 10:03:18 AM] (node:27190) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, stat '/Users/.../Sites/.../source/assets/scss/vendor/_normalize.scss'
```

**Comment by phifa** [comment]
I think I am experiencing the same isse, not sure. I am using the Foundation Zurb Framework. This is how you can reproduce: 

`npm install --global foundation-cli`
`foundation new` please choose the zurb template. 

that's it, now you can watch the files with `foundation watch`. 

Mixin and variable completion does not work, I suppose it is because of underscores/partials, etc.

**Comment by mrmlnc** [comment]
Closed in favor of #49
